### DISCUSSION TITLE
Cannot import a 3.12+ style type alias from one of my own source files?

### DISCUSSION THREAD

**Discussion by bzm3r** [microsoft/pylance-release#6728] [Q&A]
I have a source file which defines a type alias:

```
type Foo = int | tuple[int, str]
```

I'd like to import `Foo` in another file, but `pylance` gives me the error:

```
"Foo" is unknown import symbol Pylance [reportAttributeAccessIssue](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportAttributeAccessIssue)
```

How do I enable support for exporting 3.12+ style type aliases?

**Comment by erictraut** [comment]
This works fine for me. How are you importing the symbol in the second file?
